pkgs/applications/networking/irc/weechat/scripts/wee-slack/default.nix

```diff
@@ -21,11 +21,18 @@ stdenv.mkDerivation rec {
     })
   ];
 
+  postPatch = ''
+    substituteInPlace wee_slack.py --replace \
```

Comment:
Could you make this a patch instead? This way we will not silently break it, when upgrading wee slack in future. Thanks!

Reply:
Good idea! I’ve updated the PR. I believe it’s still necessary to use `substituteInPlace`, based on what I see being done in other derivations, but at least now the entire build will fail if the patch doesn’t apply cleanly.